src/pages/student/Bookmarks.css

```diff
@@ -41,6 +41,20 @@
     width: 300px;
 }
 
+.dashboard-events-photo-blob {
```

Comment:
See comments in Dashboard.js and Dashboard.scss — the same applies to the corresponding Bookmarks files.